server/server.go

```diff
@@ -1583,23 +1584,37 @@ func PrintServerAndExit() {
 	os.Exit(0)
 }
 
+// PrintBuildinfoAndExit will print out https://pkg.go.dev/runtime/debug#BuildInfo and exit.
+func PrintBuildinfoAndExit() {
+	bi, ok := debug.ReadBuildInfo()
+	if !ok || bi == nil {
+		//nolint:forbidigo
```

Comment:
@neilalexander ok to ignore like this here?

Reply:
What is the purpose of introducing this directive? After reading about the `forbidigo` tool, it flags `fmt.Print*` statements by default, however we use them elsewhere without issue.

Reply:
I added it to disable the lint warning, which was added here:
https://github.com/nats-io/nats-server/pull/3812/files

All the usages that we have are excluded in the .golangci.yml config:
```
# rg 'fmt\.Print(f|ln)?' --glob '!*_test.go'
main.go
95:     fmt.Printf("%s\n", usageStr)

server/server.go
1583:   fmt.Printf("nats-server: v%s\n", VERSION)
1592:           fmt.Println("failed to read Buildinfo")
1595:   fmt.Println(bi)

server/opts.go
4578:   fmt.Printf("%s", tlsUsage)
4580:           fmt.Printf("    %s\n", k)
4582:   fmt.Printf("\nAvailable curve preferences include:\n")
4584:           fmt.Printf("    %s\n", k)
4587:           fmt.Printf("%s\n", certstore.Usage)
4589:   fmt.Printf("%s", certidp.OCSPPeerUsage)
4590:   fmt.Printf("%s", OCSPResponseCacheUsage)
```
All other usages are in test files.

Reply:
And this
https://github.com/nats-io/nats-server/blob/cee44bef8de38b6c8a11ca6e332f998831a8e4c2/scripts/runTestsOnTravis.sh#L9-L11
will exit non-0 if there are any linter warnings, so I had to do something about it.

Reply:
Thanks for the context.